### PR TITLE
add code cleanup docs

### PR DESCRIPTION
**Pull request by terencefan**
### Summary of the changes (Less than 80 chars)
 - Detail 1
 - Detail 2

Fixes #bugnumber (in this specific format)

### CODE REVIEW

**Comment by vicancy** [comment]
Thanks for the effort. How about adding a contribution guidance in the repo, as similar to this one https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md , content here can be in the section of "How to setup develop environment"?

**Comment by terencefan** [comment]
@vicancy I think we can have this simple doc merged first and let everyone have a try on these tools.
Then decide if we could crystalize the convention into laws.

File: src/Microsoft.Azure.SignalR.Common/Endpoints/HubServiceEndpoint.cs [hunk 1]
```diff
@@ -40,11 +25,25 @@ internal class HubServiceEndpoint : ServiceEndpoint
         /// </summary>
         public Task ScaleTask => _scaleTcs?.Task ?? Task.CompletedTask;
 
+        public long UniqueIndex => _uniqueIndex;
+
+        public HubServiceEndpoint(
+                                                            string hub,
```
Comment (vicancy) [inline src/Microsoft.Azure.SignalR.Common/Endpoints/HubServiceEndpoint.cs:31]: a strange indent

**Review by vicancy — APPROVED** [review]

**Comment by terencefan** [comment]
Close since no activity

**Pull request closed without merging**